packages/dx-testing/.eslintrc.json

```diff
@@ -1,3 +1,19 @@
 {
```

Comment:
Testing package related to dx-core. It has no dependencies on frameworks

Reply:
There is './test-utils'